### ISSUE TITLE
contentView.absoluteBounds.height is not equal to device.screenHeight if statusBar.displayMode is float

### ISSUE THREAD

**Issue by fax1ty** [eclipsesource/tabris-js#1912]
### Problem description
`contentView.absoluteBounds.height` != `device.screenHeight` if `statusBar.displayMode` = `'float'`.
In my case `785.4545288085938` and  `785`

Should it be equal at all?

### Environment

- Tabris.js version: 3.0.1
- Device: Xiaomi Redmi 5 Plus
- OS: Android 8.1.0

### Code snippet

```javascript
statusBar.displayMode = 'float';
console.log(contentView.absoluteBounds.height, device.screenHeight);
```

**Comment by mpost** [comment]
Could that be due to the navigation bar taking space?

**Comment by fax1ty** [comment]
I have full screen mode enabled
![image](https://user-images.githubusercontent.com/26363214/61967447-5f6f9d80-afe6-11e9-9c0d-91cfab6a150a.png)
This feature removes the navigation bar.

But how you can see larger the height of the `contentView` and not the height of the entire `device`

**Comment by mpost** [comment]
You are right about your observation. They should both be the same number. The `device.screenHeight` is incorrectly rounded on the native side. We will fix this issue.

**Comment by mpost** [comment]
Issue is fixed in 3.1.
